components/dashboard/src/App.tsx

```diff
@@ -118,6 +119,13 @@ function App() {
     if (shouldWhatsNewShown !== isWhatsNewShown) {
         setWhatsNewShown(shouldWhatsNewShown);
     }
+    if (window.location.pathname.startsWith('/oauth-approval')) {
```

Comment:
I think this should be added below as a route

Reply:
It was originally but you end up with the menu across the top which felt out of place in the flow, allowing people to click away w/o any resolution. 

Over time we will need to add a page to list approvals and allow people to delete them, but this part of the flow is more modal.